pylint_odoo/checkers/modules_odoo.py

```diff
@@ -826,7 +826,8 @@ def _check_xml_attribute_translatable(self):
         """The xml attribute is missing the translation="off" tag
             Example  <attribute name="groups">sale.group</attribute>
         """
-        if self.config.valid_odoo_versions != ['8.0']:
+        if self.linter._all_options['valid_odoo_versions'] \
+                .config.valid_odoo_versions != ['8.0']:
```

Comment:
@moylop260 
This is la solution I found for get config for another class 
What do you think about it?

![image](https://cloud.githubusercontent.com/assets/1387970/22841601/98750558-efa8-11e6-8e19-eff7cb1bf178.png)